packages/meditrak-server/src/app/addRoutesToApp.js

```diff
@@ -250,6 +254,7 @@ export function addRoutesToApp(app) {
   app.put('(/v[0-9]+)/surveyScreenComponents/:recordId', editSurveyScreenComponents);
   app.put('(/v[0-9]+)/answers/:recordId', editAnswers);
   app.put('(/v[0-9]+)/surveyResponses/:parentRecordId/answers/:recordId', editAnswers);
+  app.put('(/v[0-9]+)/dataSources/:recordId', editDataSources);
```

Comment:
Does this work for editing data elements within the data groups dropdown (if that's possible)? The read uses something like dataSources/:dataGroupId/dataSources

Reply:
You can edit and delete via the dropdown, but those actions don't use a parent/child endpoint like the get does so this works for them

Reply:
Grand